references.py

```diff
@@ -338,5 +338,14 @@ def is_folder(self) -> bool:
             fasta='hg38/hg38.fa'
         ),
     ),
+    Source(
+        'representative_sites_table',
```

Comment:
```suggestion
        'ancestry',
```

If you want the simple config entry
```
[references.ancestry]
sites_table=gs://cpg-common-main/references/ancestry/pruned_variants.ht
```
I think this value should also be `ancestry`